modules/apps/knowledge-base/knowledge-base-api/src/main/java/com/liferay/knowledge/base/constants/KBArticleConstants.java

```diff
@@ -19,6 +21,9 @@ public class KBArticleConstants {
 
 	public static final int DEFAULT_VERSION = 1;
 
+	public static final long LOCK_EXPIRATION_TIME = GetterUtil.getLong(
+		PropsUtil.get("lock.expiration.time." + getClassName()));
```

Comment:
This is due consistency with DL logic. 

In fact, taking into account how java handles de the constants it's the same, since compiler inlines the value in this cases.